packages/contracts/contracts/marketplace/V01_Marketplace.sol

```diff
@@ -1,21 +1,47 @@
 pragma solidity ^0.4.24;
 
-import "../../node_modules/openzeppelin-solidity/contracts/ownership/Ownable.sol";
-
-/**
- * @title A Marketplace contract for managing listings, offers, payments, escrow and arbitration
- * @author Nick Poulden <nick@poulden.com>
+/*
+ * Origin Protocol
+ * https://originprotocol.com
+ *
+ * Released under the MIT license
+ * https://github.com/OriginProtocol
+ *
+ * Copyright 2019 Origin Protocol, Inc
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
  *
- * Listings may be priced in Eth or ERC20.
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
  */
 
+import "../../node_modules/openzeppelin-solidity/contracts/ownership/Ownable.sol";
+
 
 contract ERC20 {
     function transfer(address _to, uint256 _value) external returns (bool);
     function transferFrom(address _from, address _to, uint256 _value) external returns (bool);
 }
 
 
+**
```

Comment:
`/**` instead of `**`

Reply:
oh.... Doesn't seem to be a leftover `/` anywhere...  not sure what happened.  Thank you for catching that.